src/main/java/io/opdev/ExampleValuesReconciler.java

```diff
@@ -111,10 +112,25 @@ private void createFromYaml(String pathToYaml, ExampleValues resource, Context<E
   }
 
   private HasMetadata findResourceInNamespace(HasMetadata desiredObject, String namespace) {
-    ResourceDefinitionContext metaContext = new ResourceDefinitionContext.Builder()
+    String groupVersion = desiredObject.getApiVersion();
+    ResourceDefinitionContext metaContext;
+
+    if (!Strings.isNullOrEmpty(groupVersion) && groupVersion.contains("/")){
+      String group = groupVersion.split("/")[0];
+      String version = groupVersion.split("/")[1];
+      metaContext = new ResourceDefinitionContext.Builder()
+      .withGroup(group)
+      .withVersion(version)
       .withKind(desiredObject.getKind())
       .withNamespaced(true)
       .build();
+    }
+    else{
```

Comment:
There would always be a group and version right ?

Reply:
resolving as there may be a v1 without the core group specified and its possible something malformed comes through